docs/lib/snippets/modular/custom_types/table.drift.dart

```diff
@@ -319,3 +336,11 @@ class $$PeriodicRemindersTableOrderingComposer extends i0
       builder: (column, joinBuilders) =>
           i0.ColumnOrderings(column, joinBuilders: joinBuilders));
 }
+
+class $$PeriodicRemindersTableWithReferences {
```

Comment:
Instead of generating empty classes here, is it possible to define a generic class in `package:drift` and use that as a type argument to the manager for tables that don't have references? Then we don't have to generate what is essentially dead code.

Reply:
Done